### DISCUSSION TITLE
PWM Command

### DISCUSSION THREAD

**Discussion by WoodyloveBo** [bitcraze/discussions#1368] [Q&A]
hello.

I would like to know the Crazyflie flight time for each capacity of my Lipo battery by rotating the Crazyflie at maximum PWM until it discharges. I can currently issue commands to the PWM with Python scripts that ramp the motor.
https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/motors/ramp.py
So, I am curious about the two things below.

a). I am curious about how to continuously rotate the PWM to the MAX value. 
b). Looking at Mr.tobias' document, I saw that PWM commands are given in percentages from 0 to 100%. I would like to implement this.

Thank you

**Comment by gemenerik** [comment]
For a) you can look at how the reference code sends the thrust command, and rather than incrementing with thrust_step, just send your desired thrust continuously until the Crazyflie runs out of battery. I do not understand what you are asking for in question b). You want to implement what is already implemented in the reference code? Or do you want the thrust to be explicitly expressed as a percentage?
